x-pack/plugins/rollup/README.md

```diff
@@ -10,11 +13,24 @@ This plugin allows Kibana to:
 
 The rest of this doc dives into the implementation details of each of the above functionality.
 
+## Quick steps for testing
+
+The pattern for creating a rollup job and rollup index pattern is:
+
+* Install sample data (web logs is a good one).
+* Create a rollup job with an index pattern that captures this index (e.g. `k*`).
+* Set frequency to "minute". Clear the latency buffer field.
+* Select the time field which is the same time field selected in the installed index pattern (`timestamp` without an `@` in the case of web logs).
```

Comment:
Do you think it's worth mentioning the "Time bucket size" field is required?

Reply:
Good idea! Will update.